<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255)</string>
  </property>
  <widget class="QLineEdit" name="EnterLog">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EnterPass">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RepeatPass">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignUp_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 255, 127)</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>log in</string>
   </property>
  </widget>
  <widget class="QLabel" name="hint1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Enter your loggin:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hint2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Enter your password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hint3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Enter your password again:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Message">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>There will be message about errors!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="signup_background">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>381</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>signup_background</zorder>
  <zorder>EnterLog</zorder>
  <zorder>EnterPass</zorder>
  <zorder>RepeatPass</zorder>
  <zorder>SignUp_button</zorder>
  <zorder>login</zorder>
  <zorder>hint1</zorder>
  <zorder>hint2</zorder>
  <zorder>hint3</zorder>
  <zorder>Message</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
